src/go/rpk/pkg/config/manager.go

```diff
@@ -116,6 +117,15 @@ func (m *manager) ReadOrGenerate(path string) (*Config, error) {
 		path,
 	)
 	m.v.Set("config_file", path)
+	dir := filepath.Dir(path)
```

Comment:
is there any way we could test this?

Reply:
Test that `filepath.Dir` returns the dir part of the path?